packages/cardano-services/src/InMemoryCache/InMemoryCache.ts

```diff
@@ -2,7 +2,7 @@ import { CACHE_TTL_DEFAULT } from './defaults';
 import NodeCache from 'node-cache';
 
 export type Key = string | number;
-export type DbSyncQuery<T> = () => Promise<T>;
+export type Action<T> = () => Promise<T>;
```

Comment:
Could name it a little bit more specific
```suggestion
export type AsyncAction<T> = () => Promise<T>;
```

Reply:
Done in 563e7b30ca19d1de3e518e4fc451137b91d658ca